packages/ButtonGroup/stories/examples/Sandbox.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/button-has-type */
+
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { boolean, select } from "@storybook/addon-knobs";
+import { Story, Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Heading from "@paprika/heading";
+import Calendar from "@paprika/icon/lib/Calendar";
+import ButtonGroup from "../../src";
+
+const changeHandler = selectedItems => {
+  action("Selected an item")(...selectedItems);
+};
+
+const buttonGroupProps = () => ({
+  onChange: changeHandler,
+  hasIcons: boolean("hasIcons", false),
+  isDisabled: boolean("isDisabled", false),
+  isFullWidth: boolean("isFullWidth", false),
+  isMulti: boolean("isMulti", false),
+  isSemantic: boolean("isSemantic", true),
+  size: select("size", ShirtSizes.DEFAULT, ShirtSizes.MEDIUM),
+});
+
+const ExampleStory = props => (
+  <Story>
+    <Heading level={1} displayLevel={2} isLight>
+      Testing Sandbox
+    </Heading>
+    <Tagline>Use the knobs to tinker with the props.</Tagline>
+    <Rule />
+    <p>
+      <button>Pre</button>
+    </p>
+    <ButtonGroup {...props}>
+      <ButtonGroup.Item value="one" kind="primary">
+        One
+      </ButtonGroup.Item>
+      <ButtonGroup.Item value="two" isDisabled defaultIsActive>
```

Comment:
a couple more realistic stories would help see the appropriateness of the API. I.e.:
- the overall set of items can be hardcoded (e.g. "weeks/months/quarters") or dynamic (fetched from the server and coming in as an array)
- the default values can be hardcoded (e.g. always want to start with "weeks" on) or dynamic (if e.g. the last value used is stored as a preference)

I'm not sure what the actual real use cases are (the confluence page just has the one) so feel free to ignore this if you know better 👍

Reply:
As Oscar mentioned, we should have a `Docs` page with some use cases like this.  But I don't actually see any problem with the items coming from an array... that array will just need to be `map()`ed to `<ButtonGroup.Items>`.  If the selected state is included in each item, that will inform the `defaultIsActive` props.  If the selection comes separately, then something like `defaultIsActive: value === savedValue` can be added to each.

Reply:
Not gonna die on this hill for this particular component but still think those should be part of the original PR in the same spirit as https://github.com/acl-services/paprika/pull/401#issuecomment-607571444

---

```jsx
<ButtonGroup defaultValue={selectedItems}>
  {items.map((item) => (
    <ButtonGroup.Item>{item}</ButtonGroup.Item>
  ))}
</ButtonGroup>

<ButtonGroup>
  {items.map((item) => (
    <ButtonGroup.Item defaultIsActive={selectedItems.includes(item)}>{item}</ButtonGroup.Item>
  ))}
</ButtonGroup>
```
🤷‍♂️

Reply:
Yeah, that looks cleaner in the first example, but where did the `selectedItems` come from?
Maybe you'd have to do something like:
```js
const selectedItems = items.filter(item -> item.isSelected)
```
Or 
```js
const selectedItems = items.map(item -> item.value)
```
So it still seems like a toss up to me.  The reason I prefer the current approach though is that fundamentally the selected state of each item is most closely associated with that item.   

Also, if you consider `radio` or `checkbox` `<inputs>`, which this component functionally resembles, this is a closer match to that API.

Reply:
Alright, sounds good to me 👍